controllers/openstackdataplanenode_controller.go

```diff
@@ -161,7 +161,7 @@ func (r *OpenStackDataPlaneNodeReconciler) Reconcile(ctx context.Context, req ct
 		instance.Status.Conditions = condition.Conditions{}
 
 		cl := condition.CreateList(
-			condition.UnknownCondition(dataplanev1beta1.DataPlaneNodeReadyCondition, condition.InitReason, condition.InitReason),
+			condition.UnknownCondition(condition.ReadyCondition, condition.InitReason, condition.InitReason),
```

Comment:
you can remove this line, `Conditions.Init(&cl)` already adds it: https://github.com/openstack-k8s-operators/lib-common/blob/master/modules/common/condition/funcs.go#L36

Reply:
Done.